.github/workflows/build-test.yml

```
@@ -2,9 +2,9 @@ name: NodeJS build
 
 on:
   push:
-    branches: [ "main" ]
+    branches: [ "main", "dev" ]
```

yamllint (1.35.1)

[error] too many spaces inside brackets

(brackets)